assets/src/dashboard/theme.js

```diff
@@ -60,16 +65,49 @@ const theme = {
   border: {
     buttonRadius: '100px',
   },
+  dropdown: {
+    [DROPDOWN_TYPES.PANEL]: {
+      background: 'transparent',
+      activeBackground: '#f6f6f6',
+      borderRadius: '40px',
+      border: '1px solid #eee',
+      arrowColor: '#2979ff',
+    },
+    [DROPDOWN_TYPES.MENU]: {
+      background: '#f6f6f6',
+      activeBackground: '#f6f6f6',
+      borderRadius: '4px',
+      border: 'none',
+      arrowColor: '#848d96',
```

Comment:
Would it be any benefit to use the pre-defined `gray300` for this color and the other hex colors used in this dropdown section?  Just so we have one source of truth for all colors used in the dashboard?

Reply:
I wanted to do that but the theme object cannot reference stuff within itself. I can break the colors out and then reference it within theme

```
const colors = { ... }

const theme = {
 colors, ...restOfTheStuff
}
```

Reply:
i vote break out the colors, i think this is going to be really handy moving forward for things like that repeat border style too

Reply:
Yeah, I agree, breaking it out will let it be a little more flexible.